File: ExcelSheetColumnNumber/ExcelSheetColumnNumber.cpp
#include<string>
using namespace std;

class Solution {
public:
	int titleToNumber(string s) {
		int result = 0;
		for (int i = 0; i < s.size(); i++) {
			result += pow(26, i) + (s[i] - 'A');
		}
		return result;
	}
};

File: ExcelSheetColumnNumber/ExcelSheetColumnNumber.py
class Solution(object):
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        result = 0
        for i in range(len(s)):
            tmp = ord(s[i]) - 64
            result += tmp*pow(26, len(s)-1-i)
        return result